README.md

```diff
@@ -66,27 +80,30 @@ yarn start
 
 Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the contract component or the example ui in the frontend. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
 
-Run smart contract test with `yarn hardhat:test`
+Run smart contract test with `yarn hardhat:test` or `yarn foundry:test` depending of your solidity framework.
 
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
+- Edit your smart contract:
+  - Hardhat => `YourContract.sol` in `packages/hardhat/contracts`
+  - Foundry => `YourContract.sol` in `packages/foundry/contracts`
 - Edit your frontend in `packages/nextjs/pages`
-- Edit your deployment scripts in `packages/hardhat/deploy`
+- Edit your deployment scripts:
+  - Hardhat => `packages/hardhat/deploy`
+  - Foundry => `packages/foundry/script`
 
 ## Deploying your Smart Contracts to a Live Network
 
 Once you are ready to deploy your smart contracts, there are a few things you need to adjust.
 
 1. Select the network
 
-By default, `yarn deploy` will deploy the contract to the local network. You can change the defaultNetwork in `packages/hardhat/hardhat.config.ts.` You could also simply run `yarn deploy --network target_network` to deploy to another network.
+By default, `yarn deploy` will deploy the contract to the local network. You can change the defaultNetwork in `packages/hardhat/hardhat.config.ts.` or `packages/foundry/foundry.toml` depending if you are using Hardhat or Foundry. You could also simply run `yarn deploy --network target_network` to deploy to another network using Hardhat, or run `yarn deploy target_network` if you use Foundry.
```

Comment:
Once we merge #483 

The command for both are same 🙌 lets not forgot to update it